validator/src/main.rs

```diff
@@ -2227,3 +2223,8 @@ fn process_account_indexes(matches: &ArgMatches) -> AccountSecondaryIndexes {
         indexes: account_indexes,
     }
 }
+
+fn report_startup_failure_and_exit(code: i32) -> ! {
+    println!("Startup sequence failed!");
```

Comment:
Just to confirm since you may have tested this - you used `println!()` here intentionally since `eprintln!()` would get routed to log file due to this:
https://github.com/anza-xyz/agave/blob/8d5416d6563e6255646b606d92e0d266d4e7336b/validator/src/main.rs#L574

Reply:
Yes, this is intentional, so when startup fails and logs are redirected, you get a message clearly stating that it failed before validator exits.